src/crypto-api.ts

```diff
@@ -82,6 +82,15 @@ export interface CryptoApi {
      */
     exportRoomKeys(): Promise<IMegolmSessionData[]>;
 
+    /**
+     * Import a list of room keys previously exported by exportRoomKeys
+     *
+     * @param keys - a list of session export objects
+     * @param opts - progress reporting, safety and key source in the form of a ImportRoomKeysOpts
```

Comment:
I don't think we need to be quite this specific here: the place to document what's *in* an `ImportRoomKeysOpts` is in `ImportRoomKeysOpts`. 

```suggestion
     * @param opts - options object
```

likewise in MatrixClient

Reply:
done